### PR TITLE
Add draw x app

### PR DESCRIPTION
**Pull request by RoboticsYY**
This PR intends to add draw_x app.

### CODE REVIEW

File: grasp_apps/draw_x/launch/draw_x.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ur_test:
```
Comment (sharronliu) [inline grasp_apps/draw_x/launch/draw_x.yaml]: can you check why draw_x app rely on the ur_test?

File: grasp_apps/draw_x/package.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <name>draw_x</name>
+  <version>0.5.0</version>
+  <description>A demo app for draw_x</description>
+  <author email="yu.yan@intel.com">Yu Yan</author>
+  <maintainer email="yu.yan@intel.com">Yu Yan</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>Eigen3</build_depend>
```
Comment (sharronliu) [inline grasp_apps/draw_x/package.xml]: can you check if Eigen3 is necessary as build_depend?

File: grasp_apps/draw_x/launch/draw_x.yaml (hunk 1 above)
Comment (RoboticsYY) [reply grasp_apps/draw_x/launch/draw_x.yaml]: This should be the node name of the draw_x. I forgot to correct it.

File: grasp_apps/draw_x/package.xml (hunk 2 above)
Comment (RoboticsYY) [reply grasp_apps/draw_x/package.xml]: You are right. Eigen3 is not necessary here.

**Comment by sharronliu** [comment]
Looks good now, thanks!

**Pull request merged**